certification/pyxis/pyxis_suite_test.go

```diff
@@ -139,3 +140,9 @@ func (p *pyxisGraphqlLayerHandler) ServeHTTP(response http.ResponseWriter, reque
 	mustWrite(response, `{"data":{"find_images":{"error":null,"total":1,"page":0,"data":[{"uncompressed_top_layer_id":"good_top_layer","_id":"deadb33f"}]}}}`)
 	return
 }
+
+type errReader int
```

Comment:
Small comment here would help. Just something that indicates we're using this to facilitate throwing errors when passing this into io.Reader.